Makefile

```diff
@@ -497,7 +497,7 @@ bazel_test: failpoint-enable bazel_prepare
 
 
 bazel_coverage_test: failpoint-enable bazel_ci_simple_prepare
-	bazel $(BAZEL_GLOBAL_CONFIG) --nohome_rc coverage $(BAZEL_CMD_CONFIG) --jobs=35 --build_tests_only --test_keep_going=false \
+	bazel $(BAZEL_GLOBAL_CONFIG) --nohome_rc coverage -k $(BAZEL_CMD_CONFIG) --jobs=35 --build_tests_only --test_keep_going=false \
```

Comment:
What's this used for ?

Reply:
it is used to make CI continue to run since it meet errors. it will revert before merging.